docs/BackupRDB.md

```diff
@@ -0,0 +1,14 @@
+# Backup RDB guide
+
+This guide explains how to backup single RDB data with Scalar products on a Kubernetes environment. We assume that you use managed databases provided by cloud service.
+
+Even if you use RDB, you must follow the [Backup NoSQL database guide](./BackupNoSQL.md) when you use more than two RDBs under the Scalar products using [Multi-storage Transactions](https://github.com/scalar-labs/scalardb/blob/master/docs/multi-storage-transactions.md) or [Two-phase Commit Transactions](https://github.com/scalar-labs/scalardb/blob/master/docs/two-phase-commit-transactions.md).
```

Comment:
If we use more than two RDBs under ScalarDB/ScalarDL with `Multi-storage Transactions` or `Two-phase Commit Transactions`, we need to according to the manner of NoSQL backup (making restoration period by pause/unpause). I added this note here just in case.